Runtime/Model/Database/BacktraceDatabaseRecord.cs

```diff
@@ -95,7 +95,8 @@ public string BacktraceDataJson()
                 return null;
             }
 
-            return File.ReadAllText(DiagnosticDataPath);
+            DiagnosticDataJson = File.ReadAllText(DiagnosticDataPath);
+            return DiagnosticDataJson;
```

Comment:
What's the reasoning behind this change?

Reply:
On the top of this function, backtrace-unity checks if the record already cached JSON data. if it does, we return it. Otherwise, we're reading data. The problem that this change solves, allows us to cache reports even when we read it from the hard drive. This simple change can save time when someone reads data stored in the database.

Reply:
Ahhh I see, that's pretty cool!